pom.xml

```diff
@@ -225,11 +225,11 @@ flexible messaging model and an intuitive client API.</description>
     <jakarta.validation.version>2.0.2</jakarta.validation.version>
     <jna.version>5.12.1</jna.version>
     <kubernetesclient.version>12.0.1</kubernetesclient.version>
-    <okhttp3.version>4.9.3</okhttp3.version>
+    <okhttp3.version>4.10.0</okhttp3.version>
     <!-- use okio version that matches the okhttp3 version -->
-    <okio.version>2.8.0</okio.version>
-    <!-- override kotlin-stdlib used by okio in order to address CVE-2020-29582 -->
-    <kotlin-stdlib.version>1.4.32</kotlin-stdlib.version>
+    <okio.version>3.0.0</okio.version>
```

Comment:
we should move to 3.3.0, based on the [changelog](https://github.com/square/okio/blob/master/CHANGELOG.md) it's more stable and it upgrades kotlin to 1.8 as well